Pipfile

```diff
@@ -8,6 +8,8 @@ fastapi = "*"
 uvicorn = {extras = ["standard"], version = "*"}
 
 [dev-packages]
+flake8 = "*"
+autopep8 = "*"
 
 [requires]
 python_version = "3.7"
```

Comment:
Are we using 3.7? I thought we are not limited here and can use 3.10.

Reply:
For some reason `pipenv` chose by default to use that version, I'll look into changing it to 3.10.

Reply:
Changed it